src/app/components/MostRead/Canonical/List/index.tsx

```diff
@@ -9,9 +9,13 @@ import { MostReadBaseProps } from '../../types';
 const MostReadList = ({
   numberOfItems,
   dir = 'ltr',
-  columnLayout = 'multiColumn',
+  columnLayout: initialColumnLayout = 'multiColumn',
```

Comment:
I don't think we should make the changes here - I believe this logic should be in the Article page for a number of reasons:
1. separation of concerns - the article page is the only page affected by the text wrapping issue
2. blast radius - if we make the changes in the Most Read component, we would have to regression test all other affected pages, whereas if we do it in the Article page, we only need to regression test the Article Page

Reply:
The home page is also about the text wrapping issue as it is in 2 columns at the bottom too

Reply:
Hmm that's true. My only concern is by doing it in the Most Read component itself, it affects all Burmese page types and will force them all to be 1 column (although all of them now technically are - Most Read, Story Page, Article Page & HomePage)... 🤔 

OK, let's leave it as is, and if other reviewers have differing opinions on how it should work, then we can look at it then.

Reply:
I'll check if this affects the STY page at all. It is 1 column anyway, but just in case the code has done something funny with it. You are right about the regression testing being more though, sorry @paruchurisilpa !